includes/course-theme/class-sensei-course-theme-quiz.php

```diff
@@ -0,0 +1,170 @@
+<?php
+/**
+ * File containing Sensei_Course_Theme_Quiz class.
+ *
+ * @package sensei-lms
+ * @since 3.15.0
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Renders the Quiz grade results block.
```

Comment:
I think we could add a comment more generic here, since the purpose of this class would be to make all generic things with the lesson.

Reply:
Updated. 1dbac216ea54b8c0c1e61dca01c7b86edebffd8d